### ISSUE TITLE
manuscript - terminology for model outputs

### ISSUE THREAD

**Issue by elray1** [Infectious-Disease-Modeling-Hubs/hubEnsembles#44]
I've been replacing instances of "forecast" with "prediction" as an alternative noun for model outputs, but I realized that "prediction" may also be too specific. For example, we've mentioned that public health agencies might want to set a hub to collect estimates of epidemiological parameters.  Is there a more appropriate term we can use that's not just "model outputs" (which feels a bit too vague...)?

**Comment by nickreich** [comment]
I'd suggest choosing one term (either forecast or prediction seems fine to me) and in the beginning say that it is a placeholder term for any kind of model output (forecast, prediction, scenario projection, estimate, etc...) that a hub collects.

**Comment by elray1** [comment]
Makes sense.  I vote for prediction, since I think it describes forecasts and projections both fairly well.
